### DISCUSSION TITLE
Interchangeable fluid xp types

### DISCUSSION THREAD

**Discussion by Coach-mdl** [P3pp3rF1y/SophisticatedBackpacks#302] [Feature / Idea Archive] [answered]
Currently there's an issue with mob grinding utils. One can't empty the cup straight from the backpack into the tank without it being a different type of xp. While one logical method is just to remove the xp and stand on a drain, this hurts automation a little. Picture this, placing your backpack down and having it siphon both the XP and the items out of it. Amazing. I don't want to have to manually stand on the xp drain and _wait_. I'm far too busy for that! Another interesting detail is that right clicking the backpack onto the tank only puts one bucket of fluid in, and right clicking again removes the contents of the tank. I don't know if this detail is with all tanks or just the ones from mod grinding utils. Either way, i love this backpack mod.

**Comment by P3pp3rF1y** [comment]
the best would be for mob grinding utils to support standard experience fluid tag - `forge:experience` as a liquid that is experience equivalent. The Sophisticated Backpacks experience is marked with it and other mod's experience uses it as well.

**Comment by P3pp3rF1y** [comment, marked as answer]
This is all on mob grinding utils side, the xp fluid sbp uses is properly marked with the experience tag so it's up to them if they want it to go in their tank and the interaction of backpack with their tank is on their side for sure as there's no code in sbp for interaction with in world fluid handlers.
